src/atomate2/forcefields/schemas.py

```diff
@@ -131,27 +152,57 @@ def from_chgnet_result(
         final_forces = trajectory["forces"][-1].tolist()
         final_stress = trajectory["stresses"][-1].tolist()
 
-        trajectory_to_save = {key: trajectory[key] for key in keep_info}
-        if "atoms" in keep_info:
-            warning_msg = """
-                WARNING: `Atoms` objects can't be serialized (as of May 2023) and
-                thus are automatically removed.
-                """
-            logging.warning(warning_msg)
-            trajectory_to_save.pop(
-                "atoms"
-            )  # can't serialize `AseAtoms` objects, so remove them from trajectory.
-
-        steps = len(result["trajectory"])
+        n_steps = len(trajectory["energies"])
+
+        ionic_steps = []
+
+        if "keep_info" not in task_document_kwargs:
```

Comment:
@utf is this the proper way to use the `task_document_kwargs`? I feel it is not...

Reply:
No this isn't quite right. There shouldn't be a task_document_kwargs argument to this function. Add `keep_info` as an argument. When you call `ForceFieldTaskDocument.from_chgcar_result` you can do:

```python
ForceFieldTaskDocument.from_chgcar_result(structure, ..., **self.task_document_kwargs)
```

The `**` will expand task_document_kwargs (which is a dictionary). So for example, if `task_document_kwargs={"keep_info": ("energy")}`, then the above code would be equivalent to 
```python
ForceFieldTaskDocument.from_chgcar_result(structure, ..., keep_info=('energy'))
```
But python will do all the unpacking for you.

Reply:
yep, that makes a lot more sense.